client/package.json

```diff
@@ -11,7 +11,8 @@
     "lint:flow": "flow",
     "lint:eslint": "eslint 'app/**/*.js' 'app/**/*.jsx'",
     "lint:prettier": "./node_modules/.bin/prettier \"app/**/*.{js,jsx}\" --write --single-quote  --trailing-comma all",
-    "lint": "yarn lint:flow && yarn lint:prettier && yarn lint:eslint --fix",
+    "lint:stylelint": "yarn run stylelint --fix \"../**/*.scss\"",
+    "lint": "yarn lint:flow && yarn lint:prettier && lint:stylelint && yarn lint:eslint --fix",
```

Comment:
Hey @andrewhchang just one small Q: should this be `yarn lint:stylelint`?

Currently as part of the pre-commit hook, I see the following:

<img width="321" alt="Screen Shot 2020-10-05 at 9 54 01 PM" src="https://user-images.githubusercontent.com/11330558/95160425-6ac76880-0755-11eb-80cb-7a4d6d0b9080.png">

But once I added `yarn`, it worked for me 👍

Otherwise looks great!

Reply:
you're right!!! thank you - good spot!